internal/integration/api/apply-config.go

```diff
@@ -239,22 +239,18 @@ func (suite *ApplyConfigSuite) TestApplyConfigRotateEncryptionSecrets() {
 
 	encryption := machineConfig.MachineConfig.MachineSystemDiskEncryption
 
-	if encryption == nil {
+	if encryption == nil { //nolint:staticcheck
```

Comment:
golangcilint cache? recently it seems to trigger a lot of new errors, not sure if it's some corrupted cache